[12-Oct-2018 16:09:28 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('sfs', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:09:56 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Sfs" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('sfs/hello', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 16:10:03 Europe/Berlin] 
Uncaught exception: 'Error'Message: 'Call to undefined method App\Controllers\Users::test()'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/test', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 100
----------------------------------------------------------
[12-Oct-2018 16:13:24 Europe/Berlin] 
Uncaught exception: 'Error'Message: 'Call to undefined method App\Controllers\Users::test()'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/test', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 100
----------------------------------------------------------
[12-Oct-2018 16:14:34 Europe/Berlin] 
Uncaught exception: 'Error'Message: 'Call to undefined method App\Controllers\Users::all()'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 100
----------------------------------------------------------
[12-Oct-2018 16:14:38 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('hellos', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:14:56 Europe/Berlin] 
Uncaught exception: 'Error'Message: 'Call to undefined method App\Controllers\Users::index()'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/index', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 100
----------------------------------------------------------
[12-Oct-2018 16:15:23 Europe/Berlin] 
Uncaught exception: 'Error'Message: 'Call to undefined method App\Controllers\Users::index()'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/index', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 100
----------------------------------------------------------
[12-Oct-2018 16:15:41 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Hello" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('hello/index', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 16:16:20 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Hello" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('hello/index', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 16:16:40 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Hello" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('hello/index', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 16:16:44 Europe/Berlin] 
Uncaught exception: 'Error'Message: 'Call to undefined method App\Controllers\Users::index()'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/index', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 100
----------------------------------------------------------
[12-Oct-2018 16:26:26 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Method helloAction Not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\core\Router.php(100): Core\Controller->__call('hello', Array)
#1 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/hello', 'GET')
#2 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Controller.php' on line 28
----------------------------------------------------------
[12-Oct-2018 16:26:31 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Method indexAction Not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\core\Router.php(100): Core\Controller->__call('index', Array)
#1 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users/index', 'GET')
#2 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Controller.php' on line 28
----------------------------------------------------------
[12-Oct-2018 16:26:38 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('page404', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:28:50 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Sdsd" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('sdsd/sdds', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 16:29:47 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'View: home/indexs not found'
Stack trace:#0 C:\xampp\htdocs\phpmvc\app\controllers\Home.php(21): Core\View::render('home/indexs', Array)
#1 [internal function]: App\Controllers\Home->indexAction()
#2 C:\xampp\htdocs\phpmvc\core\Controller.php(30): call_user_func_array(Array, Array)
#3 C:\xampp\htdocs\phpmvc\core\Router.php(100): Core\Controller->__call('index', Array)
#4 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('', 'GET')
#5 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\View.php' on line 15
----------------------------------------------------------
[12-Oct-2018 16:35:05 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('helljgs', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:35:47 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('helljgs', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:36:12 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('helljgs', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:36:48 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'No Route match'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('sfksbf', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 83
----------------------------------------------------------
[12-Oct-2018 16:41:45 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "api\Users" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('api/users', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 16:43:28 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "Api\Users" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('api/users', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 17:00:43 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Api" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('api/users', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 17:00:50 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Method allAction Not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\core\Router.php(100): Core\Controller->__call('all', Array)
#1 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('users', 'GET')
#2 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Controller.php' on line 28
----------------------------------------------------------
[12-Oct-2018 17:04:43 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Api" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('api/users', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
[12-Oct-2018 17:07:14 Europe/Berlin] 
Uncaught exception: 'Exception'Message: 'Class "App\Controllers\Api" not exist'
Stack trace:#0 C:\xampp\htdocs\phpmvc\public\index.php(28): Core\Router->dispatch('api/users', 'GET')
#1 {main}
Throw in 'C:\xampp\htdocs\phpmvc\core\Router.php' on line 91
----------------------------------------------------------
